stories/About/Developer Guide.mdx

```diff
@@ -0,0 +1,72 @@
+import Image from "next/image";
+import '../styles.css';
```

Comment:
Can this not be in SASS? I'd prefer not to use straight CSS.

Reply:
I don't mind using SASS,  I'll install it so I can use it!